plugins/database/CMakeLists.txt

```diff
@@ -77,7 +77,6 @@ foreach(plugin IN LISTS IRODS_DATABASE_PLUGINS)
     ${plugin_target}
     PRIVATE
     "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>"
-    "$<BUILD_INTERFACE:${CMAKE_IRODS_SOURCE_DIR}/plugins/auth/include>"
```

Comment:
What motivated this change?

Reply:
Removed `#include "irods/plugins/auth/pam_password.hpp"` from `db_plugin.cpp` so we don't need this anymore (and it was a mistake to create/include that file to begin with).